framework/src/microservice.js

```diff
@@ -181,6 +187,14 @@ const Microservice = (config = {}) => {
 				.forEach(item => fnMap[type].call(this, item)));
 	};
 
+	const scheduleJobs = async () => {
```

Comment:
Please move `scheduleJobs` above `_addItems`. Kindly implement the suggested change and republish the framework with the same version `1.4.5`.

Also, please update the dependencies in all the microservices wherever applicable so that we can test the changes.

Reply:
Applied